packages/dapp/src/components/listinglist/WhitelistedListingListRedux.tsx

```diff
@@ -25,7 +27,16 @@ const WhitelistedListingListRedux: React.SFC<WhitelistedListingsListReduxReduxPr
     const unchallengedListings = props.whitelistedListings.filterNot(predicate).toSet();
     const groupedListings = challengedListings.concat(unchallengedListings).toSet();
 
-    return <ListingList ListingItemComponent={ListingSummaryApprovedComponent} listings={groupedListings} />;
+    return (
+      <>
+        <StyledListingCopy>
+          All approved Newsrooms agreed to uphold the journalistic principles in the{" "}
+          <a href="https://civil.co/constitution/">Civil Constitution</a>, and Newsrooms are subject to Civil's{" "}
+          <a href="#zendesk">community vetting process</a>.
```

Comment:
Should this be https://cvlconsensys.zendesk.com/hc/en-us instead of `#zendesk`?

Reply:
These will eventually link to the actual FAQ pages in Zendesk, but those don't exist yet.